app/services/fallback/__init__.py

```
@@ -0,0 +1,286 @@
+                failure_threshold=await BB_STT_FALLBACK_THRESHOLD(),
+                failure_window_secs=await BB_STT_FALLBACK_WINDOW_SECS(),
+                fallback_duration_secs=await BB_STT_FALLBACK_DURATION_SECS(),
+                fallback_provider_name=fallback_provider,
+            )
+        )
+        if not await fallback.is_active():
+            return
+
+        logger.info("STT fallback active — resetting to primary provider")
+        await fallback.reset_to_primary()
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)